src/Symfony/Component/Intl/Tests/CountriesTest.php

```diff
@@ -646,6 +657,8 @@ public function testGetAlpha3Name($displayLocale)
 
     public function testGetAlpha3NameWithInvalidCountryCode()
     {
+        IntlTestHelper::requireFullIntl($this);
```

Comment:
does this actually require a full intl ?

Reply:
indeed, it's not required here